config/v1/0000_10_config-operator_01_helmchartrepository.crd.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helmchartrepositories.config.openshift.io
+spec:
+  scope: Cluster
+  preserveUnknownFields: false
+  group: config.openshift.io
+  names:
+    kind: HelmChartRepository
+    listKind: HelmChartRepositoryList
+    plural: helmchartrepositories
+    singular: helmchartrepository
+  versions:
+  - name: v1
```

Comment:
General question: should we call it v1 right away?

Reply:
Good catch, I think it should `v1alpha1`

Reply:
moved to v1beta1